projects/storefrontlib/src/cms-components/configurator/commons/config-attribute-types/config-attribute-checkbox-list/config-attribute-checkbox-list.component.html

```diff
@@ -32,8 +32,15 @@
           )
         }}"
         class="cx-config-attribute-value-label form-check-label"
-        >{{ value.valueDisplay }}</label
       >
+        <ng-container *ngIf="attribute.isNumeric">
+          {{ value.valueDisplay }}
+        </ng-container>
+
+        <ng-container *ngIf="!attribute.isNumeric">
```

Comment:
Why 2 containers with the same content?

Reply:
Checked it again: This is still wrong, correct? Because those containers do the very same thing. Did you mean to put the pipe here?